Adding information about how to add a hidden folder (#1314)
platformsh/platformsh-docs: src/configuration/app/storage.md

See the [web locations](/configuration/app/web.md) documentation for more details.

> ## Unable to mount hidden folder?

Reviewer: ```suggestion
## Why can't I mount a hidden folder?
```

> We ignore every YAML key that starts with a dot. This causes a mount like '.myhiddenfolder' to be ignored. If you want to mount a hidden folder, you'll have to prepend it with a `/`

Reviewer 2: ```suggestion
Platform.sh ignores YAML keys that start with a dot. This causes a mount like `.myhiddenfolder` to be ignored. If you want to mount a hidden folder, you'll have to prepend it with a `/`:
```

```yaml
mounts:
  '/.myhiddenfolder':
    source: local

>     source_path: '.myhiddenfolder'

Reviewer 2: ```suggestion
    source_path: 'myhiddenfolder'
```

Reviewer 2: This one's your call. In our templates we don't keep mount source paths hidden to keep it simple and visible, so this suggestion is more of keeping it consistent with that choice.

Reviewer: Dot files are annoying, so let's not use them in source paths.
